lambda-slack.tf

```diff
@@ -0,0 +1,67 @@
+resource "aws_iam_role" "lambda_role" {
+name   = "slack-lambda-role"
```

Comment:
Could you add a flag variable, e.g. `alerts_enabled` to enable/disable the alert related resources?